src/App.tsx

```diff
@@ -2,108 +1,74 @@
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
+const getItemNum = (item: string): number => {
+  const idx = item.indexOf(' ');
+  return +item.slice(idx);
```

Comment:
The `getItemNum` function assumes that the item string always contains a space followed by a number. If the format of the item string changes, this function may not work correctly. Consider adding validation or error handling to ensure the string format is as expected.